db/migrate/20170607001253_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.string :comment
```

Comment:
I would recommend naming this attribute something other than the name of the model itself (e.g., body, text).